modules/indexer/code/indexer.go

```diff
@@ -205,9 +202,6 @@ func Init() {
 
 			rIndexer, populate, err = NewElasticSearchIndexer(setting.Indexer.RepoConnStr, setting.Indexer.RepoIndexerName)
```

Comment:
We may need to `Close` inside `NewElasticSearchIndexer` like `NewBleveIndexer`

Reply:
Done.